src/WebJobs.Script.Abstractions/WebJobs.Script.Abstractions.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Need to add property to use "Microsoft.Azure.WebJobs.Script.Abstractions" for dll name